Add 'clipboard' to the default features so we don't have to maintain
d/control manually
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -47,7 +47,7 @@
 
 [features]
 clipboard = ["terminal-clipboard"]
-default = []
+default = ["clipboard"]
 kitty-csi-check = ["xterm-query"]
 
 [lib]
